erpnext/crm/doctype/opportunity/opportunity.js

```diff
@@ -3,6 +3,7 @@
 
 {% include 'erpnext/selling/sales_common.js' %}
 {% include 'erpnext/selling/quotation_common.js' %}
+{% include 'erpnext/selling/applies_to_common.js' %}
```

Comment:
Where's the Vehicles domain installer? How about we move this line to the other branch?